pkg/auth/providers/keycloakoidc/keycloak_client.go

```diff
@@ -184,31 +181,31 @@ func URLEncoded(str string) string {
 	return u.String()
 }
 
-func (k *KeyCloakClient) getFromKeyCloak(url string) ([]byte, int, error) {
+func (k *KeyCloakClient) getFromKeyCloak(url string) ([]byte, error) {
 	req, err := http.NewRequest("GET", url, nil)
 	if err != nil {
-		return nil, 500, err
+		return nil, err
 	}
 	req.Header.Add("Accept", "application/json")
 	resp, err := k.httpClient.Do(req)
 	if err != nil {
 		logrus.Errorf("[keycloak oidc]: received error from keycloak: %v", err)
-		return nil, resp.StatusCode, err
+		return nil, err
 	}
 	defer resp.Body.Close()
 	b, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
-		return b, resp.StatusCode, err
+		return b, err
 	}
 	switch resp.StatusCode {
 	case 200:
```

Comment:
why have these if they are handled by the default case?

Reply:
This error catch is for the errors with the ReadAll function. The switch uses the response code, if this error catch was not here, it would be possible for the ReadAll function to return an error because it had issues reading the body, but the response code could be 200, in which case the error would get eaten.